components/snackbar.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-snackbar v-model="open" :timeout="3000" :color="variant">
+    <span :style="{ color: textColor }">{{ message }}</span>
+    <v-btn :style="{ color: textColor }" text @click="open = false"
+      >Close</v-btn
+    >
+  </v-snackbar>
+</template>
+
+<script lang="ts">
+export default {
+  data() {
+    return {
+      open: false,
+      variant: '',
+      message: '',
+    }
+  },
+  computed: {
+    textColor: function () {
+      const variantToTextColorMap = {
```

Comment:
I really wish that Vuetify's theme customization was a bit more flexible and also allowed to specify which text color to use for each of the theme's variants (the Material-ui library that I normally use in React does that, and it's quite handy! 😄). I guess this works for now, if we need to re-use these somewhere else we can always figure out something then.

Reply:
Ah yeah, it's too bad that is not supported at the moment, but glad that you were able to work around that! Maybe it will be supported by vuetify in the future.